erdpy/contract_verification.py

```diff
@@ -13,22 +13,20 @@ def __init__(
         contract: Address,
         source_code: Dict[str, Any],
         signature: Any,
-        docker_tag: str,
+        docker_image: str,
     ) -> None:
         self.contract = contract
         self.source_code = source_code
         self.signature = signature
-        self.docker_tag = docker_tag
+        self.docker_image = docker_image
 
     def to_dictionary(self) -> Dict[str, Any]:
-        contract_verification_request: Dict[str, Any] = {}
-
-        contract_verification_request["contract"] = self.contract
-        contract_verification_request["source_code"] = self.source_code
-        contract_verification_request["signature"] = self.signature
-        contract_verification_request["docker_tag"] = self.docker_tag
-
-        return contract_verification_request
+        return {
+            "contract": self.contract,
+            "source_code": self.source_code,
+            "signature": self.signature,
+            "docker_tag": self.docker_image
```

Comment:
The dictionary key can be `dockerImage` (JSON-friendly). Also, `sourceCode` (cammel case).

Reply:
Done